IDEAS/Buildings/Data/Interfaces/Glazing.mo

```diff
@@ -55,10 +56,18 @@ to reflect the effect of the glazing coating, assuming it exists.
 This makes the difference of a U value of about 3 W/mK or 1 W/mK for double glazing!
 See <a href=\"modelica://IDEAS.Buildings.Data.Glazing.Ins2Ar\">IDEAS.Buildings.Data.Glazing.Ins2Ar</a>
 for an example with double glazing that has 1 glazed sheet with a coating on the inside.
+<p> When no coatings are detected, ie. when the emissivities are default, an error is thrown to inform 
```

Comment:
You add the start of a paragraph using `<p>` but not its end. Also, you inserted a paragraph within another paragraph.